color_segmenter/ColorBlobDescriptor blob_descriptor
---
color_segmenter/ColorBlobArray blob_array
